winforms/src/toga_winforms/widgets/base.py

```diff
@@ -37,17 +40,18 @@ def scale_round(self, value, rounding):
 
 
 class Widget(ABC, Scalable):
-    # In some widgets, attempting to set a background color with any alpha value other
-    # than 1 raises "System.ArgumentException: Control does not support transparent
-    # background colors". Those widgets should set this attribute to False.
-    _background_supports_alpha = True
 
     def __init__(self, interface):
         self.interface = interface
         self.interface._impl = self
 
         self._container = None
         self.native = None
+
+        # Widgets that need to set a different default background_color
+        # should override this attribute.
+        self._default_background_color = toga_color(SystemColors.Control)
```

Comment:
Based on the number of files that needed to change, there's possibly an argument to be made that the default value here should be TRANSPARENT, with widget-specific colors set on widgets that don't fall back to transparent.

Reply:
I'm afraid making TRANSPARENT as the default would unnecessarily manifest visibility bugs in some widgets - as previously encountered while comparing with webview widgets. 

Moreover, there only 5-6 widgets on which the default background color is TRANSPARENT. If we make TRANSPARENT as the default, then there would be a greater number of widgets that would be setting a non-TRANSPARENT color as the default background. Hence, the number of changed files would be even more.

Reply:
OK - that makes sense.